images/epg-help.xpm

```diff
@@ -1,248 +1,257 @@
 /* XPM */
-static char * epg_help_xpm[] = {
-"24 24 221 2",
+static char * help_xpm[] = {
```

Comment:
Probably it should be changed (not sure if it makes a difference). 

The name is an artifact of how I created the file.